### PR TITLE
fix(service-worker): add missing annotation for SwPush

### PR DESCRIPTION
**Pull request by trotyl**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #19525 

## What is the new behavior?

Service worker being available in JIT usage.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

This is just adding a static annotation, do we need integration test to cover it?

### CODE REVIEW

**Review by gkalpak — COMMENTED** [review]
The change LGTM. Not sure about testing. Wouldn't a simple unit test cover that? I think it would be nice to have some unit tests that access the services via DI anyway.

**Comment by trotyl** [comment]
Added a test for DI access.

File: packages/service-worker/test/comm_spec.ts [hunk 1]
```diff
@@ -63,6 +64,15 @@ export function main() {
...
         });
       });
+      it('is injectable', () => {
+        TestBed.configureTestingModule({
+          providers: [
+            SwPush,
+            {provide: NgswCommChannel, useValue: comm},
+          ]
+        });
+        expect(() => TestBed.get(SwPush)).not.toThrow();
```
Comment (gkalpak) [inline packages/service-worker/test/comm_spec.ts]: Could you add a similar test for `SwUpdate`? (And while you are at it, maybe update the descriptions: `describe('NgswPush'/'NgswUpdate'` --> `describe('SwPush'/'SwUpdate'` :grin:)

Comment (trotyl) [reply packages/service-worker/test/comm_spec.ts]: Done~ 😃

**Review by gkalpak — APPROVED** [review]

**Pull request closed without merging**